frontend/src/entry/model/settings/ModelAccess.tsx

```diff
@@ -142,9 +145,19 @@ export default function ModelAccess({ model }: ModelAccessProps) {
         </TableHead>
         <TableBody>{accessListEntities}</TableBody>
       </Table>
-      <LoadingButton variant='contained' aria-label='Save access list' onClick={updateAccessList} loading={loading}>
-        Save
-      </LoadingButton>
+      <Tooltip title={roleError}>
```

Comment:
This `Tooltip` and `span` won't be required if you use `errorMessage` to display the error.